html/index.njk

```diff
@@ -63,10 +63,28 @@
                     E-mail: <a href="mailto:adela@ticha.fun">adela@ticha.fun</a>
                 </p>
             </div>
-            <div class="col-12 col-md-6">
-                <img class="img-fluid d-none d-md-inline shadow-lg rounded-lg" src="../images/profile-picture-2-1000x850.jpg" alt="My Photo" width="500" height="425">
-                <img class="img-fluid d-md-none shadow rounded" src="../images/profile-picture-2-680x578.jpg" alt="My Photo" width="340" height="298">
-            </div>
+			<div class="col-12 col-md-6 mt-3">
+				<p>
+					If you'd like me to get back to you, please send me a message. If you'll describe your needs in the message, we can talk specifically.
+				</p>
+				<form>
```

Comment:
I see. I think I know what you mean, just thinking where to define the variables. In` gulpfile.js`?

Reply:
Oh, I see now. Just added it here, is that OK like that?

Once we know the URLs I know we have to add it to gulpfile.js in infra repository.

Reply:
Well, you will need to define them in the gulpfile to have some values for local development, however, you will need to add them to the deployment process as well.

Reply:
So I guess it will look like that, right? Now we need to get the URLs
![image](https://user-images.githubusercontent.com/57351253/180795032-affe9bb7-83d5-4ed9-981d-e1a122c0d529.png)

Reply:
I see. Created PR in the Infrastructure repository for it with a nonsense URL.